plugins/woocommerce-blocks/assets/js/blocks/coming-soon/entire-site.scss

```diff
@@ -202,3 +202,22 @@ body:has(.woocommerce-coming-soon-banner) {
 		aspect-ratio: 1 / 1;
 	}
 }
+
+// Product image gallery template styles
+.woocommerce-coming-soon-image-gallery {
+	.wp-block-loginout {
+		display: flex;
+
+		a {
```

Comment:
We should remove these custom CSS styles and use the block settings to control the styles instead so that users can adjust the styles in block editor.

https://github.com/user-attachments/assets/c7176ad6-1c5b-4965-8d57-f3591ecb749f